CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.5)
+cmake_minimum_required(VERSION 3.14) # default install paths
 project(QuaZip)
 
 # CMP0042: Explicitly acknowledge MACOSX_RPATH
```

Comment:
consider https://cmake.org/cmake/help/latest/command/cmake_policy.html#setting-policies-by-cmake-version to set everything to new. Usually the newer policies are better:
`cmake_policy(VERSION 3.14.0)`